zerver/views/tutorial.py

```diff
@@ -1,15 +1,15 @@
 from django.http import HttpRequest, HttpResponse
 
 from zerver.decorator import human_users_only
-from zerver.lib.request import REQ, has_request_variables
 from zerver.lib.response import json_success
+from zerver.lib.typed_endpoint import typed_endpoint
 from zerver.models import UserProfile
 
 
 @human_users_only
-@has_request_variables
+@typed_endpoint
 def set_tutorial_status(
-    request: HttpRequest, user_profile: UserProfile, status: str = REQ()
+    request: HttpRequest, user_profile: UserProfile, *, status: str
```

Comment:
This should probably be converted to a Literal of the couple valid strings.

Reply:
I've made the changes